Mixpanel/SelectorEvaluator.h

```diff
@@ -0,0 +1,101 @@
+//
+//  SelectorEvaluator.h
+//  Mixpanel
+//
+//  Created by Madhu Palani on 3/15/19.
+//  Copyright © 2019 Mixpanel. All rights reserved.
+//
+#import <Foundation/Foundation.h>
+@interface MPBoolean : NSObject
+
+@property (nonatomic, readonly) BOOL value;
+
+- (instancetype)init:(BOOL)value;
+
+@end
+
+@interface SelectorEvaluator : NSObject
++(NSDate*) getCurrentDate;
++(NSDateFormatter*) getDateFormatter;
```

Comment:
nit: + (NSDateFormatter *)getDateFormatter;
also in objective c, for naming methods, normally we don't use `get` here.
https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html